### ISSUE TITLE
gen 6 mystery gift legality error

### ISSUE THREAD

**Issue by NinFanBoyFTW** [kwsch/PKHeX#1400]
Hi. I was checking my pokemon omega ruby save and I saw that my event sammurott that came with its hidden ability was flagged as illegal. I havent touched it at all and when I exported the wc6 and imported it as a pokemon in tabs it still says its illegal
[Card # 1507 - It's Samurott!.zip](https://github.com/kwsch/PKHeX/files/1233231/Card.1507.-.It.s.Samurott.zip)

**Comment by kwsch** [comment]
Fixed gender event isn't in event database
@kamronbatman

**Comment by NinFanBoyFTW** [comment]
I just wanted to let you know I looked through my other events and it looks like the same thing is happening with my Zoroark event. here is the pk6 and the wc6
[571 - Zoroark - 95B9E12D7B70.zip](https://github.com/kwsch/PKHeX/files/1233464/571.-.Zoroark.-.95B9E12D7B70.zip)

**Comment by kamronbatman** [comment]
Looks like we were missing a few genders.  @NinFanBoyFTW, please give me a list of the ones you get errors for and I'll fix it ASAP.

**Comment by kamronbatman** [comment]
I added the missing WCs.  Please let me know if there are any others!

**Comment by NinFanBoyFTW** [comment]
Hi. my samurott and zoroark are still showing up as illegal. I just compiled. so im not sure

**Comment by NinFanBoyFTW** [comment]
Ok I just found one more in my pokemon sun and moon. The new sallazle event that just came out is saying illegal
[salazzle.zip](https://github.com/kwsch/PKHeX/files/1236057/salazzle.zip)

**Comment by kamronbatman** [comment]
We got salazzle from the servers. It is possible PKHex doesn't have the newest database and dynamic updates are not available yet.  I recommend using the Event gallery directly and dumping the event files into a folder called `mgdb` in the same directly as PKHex.

**Comment by NinFanBoyFTW** [comment]
ok so I dumped the event database to that folder and it still didnt work. When I put myt wc6 in this folder it then showed that the pokemon were legal but obviously this is still a problem since I had to put my own wc6 in a manual database. I dont understand what the problem is with these wc6. Do you know why it is like this. I have the exact same events on my pokemon alpha sapphire and the pokemon that come from those wc6 are legal

**Comment by kwsch** [comment]
Certain events are available with multiple genders; however, once downloaded, the wc6/7 becomes a fixed gender.

Basically just need to add the gift to the event database.
Might be easier to scan through the database to find split-gender events... it'd only need 1 byte changed  (gender) for a 100% legal copy.

**Comment by NinFanBoyFTW** [comment]
ok I think I understand now. I didnt realize the ones I had trouble with were both female and those species of pokemon had 12.5% change of being female so I guess I got pretty lucky with these events then. I still dont understand why my salazzle from the event that is going on right now is showing up as illegal. I checked the event database for sun and moon and it is showing up that you guys have it so clearly is isnt a problem with the database not updating. Also this cant be the same issue as the other ones then because salazzle is only female.
